xdcs-frontend/src/app/view/task-creation/task-creation.component.ts

```diff
@@ -67,6 +82,16 @@ export class TaskCreationComponent {
       });
   }
 
+  setValuesForReservedParams() {
+    if (!!this.deployment.config.kernelparams) {
+      Array.from(this.deployment.config.kernelparams).forEach(param => {
+        if (this.reservedParamNames.includes(param.name)) {
+          this.kernelArguments[param.name] = 'reserved';
```

Comment:
skąd ta wartość `reserved` i co ona oznacza?

Reply:
to jest stały string tylko po to, żeby wszystkie argumenty traktować tak samo, żeby miały jakąkolwiek wartość w OR, przed uruchomieniem kernela ta wartość jest nadpisywana zależnie od nazwy parametru

Reply:
czyli to się zapisze w OR?

Reply:
tak